onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -95,10 +95,13 @@ def aten_addmm(
 ) -> TFloat:
     # addmm(Tensor self, Tensor mat1, Tensor mat2, *, Scalar beta=1, Scalar alpha=1) -> Tensor
 
+    # TODO(titaiwang): op.Gemm seems needed to take care of corner case according to old symbolic_fn.
+    # Currently, it shows op-level validation failing on bloom.
```

Comment:
What were the cases? Would replacing the body with Gemm make more sense?

Reply:
The idea comes from https://github.com/pytorch/pytorch/pull/46570/files#diff-6616e68665d29ba8ab30d14fc99a8459f974fb58368d71daf7bca89701e557db. But it seems that even though we don't use Gemm, we should still be fine. THe op-level validation is failing with mismatch rn.

Reply:
Track by https://github.com/microsoft/onnx-script/issues/424